python/Ganga/GPIDev/Schema/Schema.py

```diff
@@ -159,8 +160,8 @@ def inherit_copy(self):
     def createDefaultConfig(self):
         # create a configuration unit for default values of object properties
         # take the defaults from schema defaults
-        config = Ganga.Utility.Config.makeConfig(defaultConfigSectionName(self.name),
-                                                 "default attribute values for %s objects" % self.name)
+        _self_name = self.name
```

Comment:
to avoid calling self.name twice which is more than just getting an attribute due to some changes.